static/src/javascripts/projects/common/modules/commercial/dfp/dfp-api.js

```diff
@@ -407,7 +407,9 @@ define([
     }
 
     function shouldRenderLabel($adSlot) {
-        return $adSlot.data('label') !== false && qwery('.ad-slot__label', $adSlot[0]).length === 0;
+        return !$adSlot.hasClass('ad-slot--frame') &&
```

Comment:
Perhaps via a config object somewhere? The thing is more complicated than it looks, some creative I saw already had a label, which presumably is why this function existed in the first place.